### ISSUE TITLE
Skip the directory under "contracts" without Cargo.toml

### ISSUE THREAD

**Issue by h4x3rotab** [l00k/devphase#32]
Now if there's an empty directory under `contracts/`, devphase will raise an error:

```
[06:30:59] Compilation [started]
ERROR: Loading Cargo.toml

Caused by:
    No such file or directory (os error 2)
```

It would be nice if it can automatically skip the non-rust directory.

Background: This is useful when I'm working on a git managed project. I may switch from a newer commit to an older commit, but the source code I added in the newer commit is deleted in the working tree while there are still a few files left inside (e.g. `.env` file with some API keys). I want to finish the work on the old commit, and switch back to then new commit later. In this case, I have a directory left under `contracts/` without the real source code, but I would like to keep it.

**Comment by l00k** [comment]
Absolutetly right. Fixed `v0.6.2`

**Comment by h4x3rotab** [comment]
Nice!
